plugins/modules/vmware_dvs_host.py

```diff
@@ -126,17 +158,32 @@
 class VMwareDvsHost(PyVmomi):
     def __init__(self, module):
         super(VMwareDvsHost, self).__init__(module)
-        self.dv_switch = None
         self.uplink_portgroup = None
         self.host = None
         self.dv_switch = None
-        self.nic = None
+        self.desired_state = {}
+
         self.state = self.module.params['state']
         self.switch_name = self.module.params['switch_name']
         self.esxi_hostname = self.module.params['esxi_hostname']
         self.vmnics = self.module.params['vmnics']
+        self.lag_uplinks = self.module.params['lag_uplinks']
         self.vendor_specific_config = self.module.params['vendor_specific_config']
 
+        self.dv_switch = find_dvs_by_name(self.content, self.switch_name)
+
+        if self.dv_switch is None:
+            self.module.fail_json(msg="A distributed virtual switch %s "
+                                      "does not exist" % self.switch_name)
+
+        self.lags = {}
+        for lag in self.dv_switch.config.lacpGroupConfig:
+            self.lags[lag.name] = lag
+
+        for lag_uplink in self.lag_uplinks:
+            if lag_uplink['lag'] not in self.lags:
+                self.module.fail_json(msg="LAG %s not found" % lag_uplink.lag)
```

Comment:
```suggestion
                self.module.fail_json(msg="LAG %s not found" % lag_uplink['lag'])
```

Reply:
Thanks, @mariolenz for the patch!

Rest LGTM.  
The following error has occurred if the lag doesn't exist.

```
The full traceback is:
Traceback (most recent call last):
(snip) 
  File "/usr/local/Cellar/python@3.9/3.9.1_1/Frameworks/Python.framework/Versions/3.9/lib/python3.9/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/var/folders/_k/nxbxmsc55r5gkfm273bnr1xc0000gn/T/ansible_vmware_dvs_host_payload_newelp7y/ansible_vmware_dvs_host_payload.zip/ansible/modules/vmware_dvs_host.py", line 414, in <module>
  File "/var/folders/_k/nxbxmsc55r5gkfm273bnr1xc0000gn/T/ansible_vmware_dvs_host_payload_newelp7y/ansible_vmware_dvs_host_payload.zip/ansible/modules/vmware_dvs_host.py", line 409, in main
  File "/var/folders/_k/nxbxmsc55r5gkfm273bnr1xc0000gn/T/ansible_vmware_dvs_host_payload_newelp7y/ansible_vmware_dvs_host_payload.zip/ansible/modules/vmware_dvs_host.py", line 185, in __init__
AttributeError: 'dict' object has no attribute 'lag'
```

Reply:
Thx @sky-joker!